rodanandfields-clone/src/Components/Consultant.jsx

```diff
@@ -100,24 +114,362 @@ const Consultant = () => {
+                  and share content about your business and products with ease.
+                  • The full R+F Virtual site for exclusive training videos and
+                  analysis tracking as you share with prospective business
+                  partners. • Free shipping on all orders of 100 Sales Volume
+                  (SV) or more. That’s just a sample of what is available with
+                  PULSE by Penny Pro. To see the full breakdown between Pro and
+                  Basic, please see the PULSE by Penny Comparison Chart.
+                </AccordionPanel>
+              </AccordionItem>
+
+              <AccordionItem>
+                <h2>
+                  <AccordionButton>
+                    <Box p="12px 0px" flex="1" textAlign="left">
+                      <Text fontSize="17px">What is CRP?</Text>
+                    </Box>
+                    <AccordionIcon />
+                  </AccordionButton>
+                </h2>
+                <AccordionPanel pb={4}>
+                  The Consultant Replenishment Program (CRP) is an optional
+                  program offered to Consultants to simplify the process of
+                  ordering R+F products. Subscribing to CRP means you will
+                  receive regular monthly shipments of your chosen products at
+                  your special Consultant price. Your credit card is
+                  automatically charged each time. You can easily cancel or edit
+                  your CRP order anytime.
+                </AccordionPanel>
+              </AccordionItem>
+
+              <AccordionItem>
+                <h2>
+                  <AccordionButton>
+                    <Box p="12px 0px" flex="1" textAlign="left">
+                      <Text fontSize="17px"> What’s the VIP Box?</Text>
+                    </Box>
+                    <AccordionIcon />
+                  </AccordionButton>
+                </h2>
+                <AccordionPanel pb={4}>
+                  The Very Important Product (VIP) Box is a one-time offer
+                  allowing new Consultants to order personalized products at an
+                  additional 30% off the Consultant price. Though the discount
+                  is good for up to 60 days after enrollment, if ordered on the
+                  same day as enrollment, complimentary two-day shipping is
+                  included. This is the best way to experience our amazing
+                  products at an incredible value.
+                </AccordionPanel>
+              </AccordionItem>
+            </Accordion>
+          </Box>
+          <Box m="60px 350px">
+            <Button
+              bg="#F8F8F8"
+              borderRadius="3px"
+              p="15px 40px"
+              border="1px solid black"
+              _hover="none"
+            >
+              SEE MORE
+            </Button>
+          </Box>
+        </Box>
+      </Box>
```

Comment:
component is 400+ lines. We can make a constant file and render AccordionPanel data while looping. we have hard-coded lot of things